Automattic/wp-calypso
# Theme showcase: Update the layout on the Global Site View

Related to https://github.com/Automattic/wp-calypso/issues/94950

## Proposed Changes

* Update the layout of the Theme showcase (header and theme content)

## Why are these changes being made?


* This PR makes the themes layout more consistent with the rest of the sections (Sites and P2)

## Testing Instructions



* Use the calypso live link
* Go to /sites and then /themes global pages
* Notice that the "Themes" and the "Design your own" elements are aligned with the rest of the global pages (Sites is leading)
* Make sure that no regressions are affecting /themes/site-name and the logged out showcase

| Before  | After |
| ------------- | ------------- |
![Screenshot 2024-10-22 at 12 52 40](https://github.com/user-attachments/assets/f6c4e2d7-ab93-4f3f-9e02-07e668032db2)  | ![Screenshot 2024-10-22 at 12 51 12](https://github.com/user-attachments/assets/9fe761c0-0817-49be-a70f-001d779f5d85)  |
 ![Screenshot 2024-10-22 at 12 52 10](https://github.com/user-attachments/assets/ccb3b55a-f486-4b94-9322-d9f6c0829fd5) | ![Screenshot 2024-10-22 at 13 06 56](https://github.com/user-attachments/assets/9c34b047-4b41-4b1b-bed3-9de9d88599a0) |


## Pre-merge Checklist
